variables.tf

```diff
@@ -72,7 +77,7 @@ variable "etcd_addresses" {
 
 variable "etcd_instance_type" {
   default     = "t2.small"
-  description = "The type of etcd instances to launch."
+  description = "The type of etcd instances to launch. Must be of the t2 family"
```

Comment:
just out of curiosity, will terraform fail if we pass something different than `t2`? or is this just as a deterrent for user to avoid other inputs?

Reply:
User information. Terraform will apply the changes but the volumes would never get mounted in the machines.

Reply:
*Could* we somehow fail on a non-`t2` family type?